bigquery/tests/system.py

```diff
@@ -1082,14 +1090,16 @@ def test_sync_query_w_query_params(self):
             },
         ]
         for example in examples:
-            query = Config.CLIENT.run_sync_query(
+            query_job = Config.CLIENT.run_async_query(
+                'test_query_w_query_params{}'.format(unique_resource_id()),
                 example['sql'],
                 query_parameters=example['query_parameters'])
-            query.use_legacy_sql = False
-            query.run()
-            self.assertEqual(len(query.rows), 1)
-            self.assertEqual(len(query.rows[0]), 1)
-            self.assertEqual(query.rows[0][0], example['expected'])
+            query_job.use_legacy_sql = False
```

Comment:
Shouldn't be necessary: using query params should force this internally.

Reply:
I tried removing and got "google.api.core.exceptions.BadRequest: 400 POST https://www.googleapis.com/bigquery/v2/projects/swast-scratch/jobs: Cannot use query parameters in a legacy SQL query."

I'll investigate further when I replace `run_async_query`.